.github/workflows/submodule.yaml

```diff
@@ -46,3 +46,37 @@ jobs:
         git_hash="$(git rev-parse --short :web-client)"
         git commit -S -m "build(web-client): update submodule to $git_hash" || echo "No changes to commit"
         git push
+
+  update-cryostat3-submodule:
+    if: ${{ github.repository_owner == 'cryostatio' }}
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v4
+      with:
+        repository: cryostatio/cryostat3
+        token: "${{ secrets.SUBMODULE_TOKEN }}"
+        ref: "${{ github.ref_name }}"
+    - name: Check remote cryostat3 submodule branch
+      run: |
+        remote_branch="$(git config --get -f .gitmodules submodule.webui.branch)"
```

Comment:
Looks like this line probably doesn't work:

```bash
$ git config --get -f .gitmodules submodule.webui.branch) # no output
$ git config --get -f .gitmodules submodule."src/main/webui".url
https://github.com/cryostatio/cryostat-web
```

The `.gitmodules` file in the cryostat3 repo looks like this:

```
[submodule "src/main/webui"]
	path = src/main/webui
	url = https://github.com/cryostatio/cryostat-web
```

whereas in the old repo, it is:

```
$ cat .gitmodules 
[submodule "web-client"]
	path = web-client
	url = https://github.com/cryostatio/cryostat-web
	branch = main
```

So,
1. either the submodule needs to be renamed in the `.gitmodules` file, or this line needs to replace `submodule.webui` with `submodule."src/main/webui"`
2. the `branch` key needs to be added to the cryostat3 `.gitmodules` file